week3/homework/src/index.js

```diff
@@ -1,3 +1,26 @@
-'use strict';
+"use strict";
 
-// TODO: Write the homework code in this file
+const fs = require("fs");
+const express = require("express");
+const bodyParser = require("body-parser");
+const app = express();
+const myPackage = require("./myPackage");
+app.use(bodyParser.json());
+
+app.get("/todos/:id", (request, response) => {
+  myPackage.readToDo(request, response);
+});
+
+app.delete("/todos/", (request, response) => {
+  myPackage.clearToDos(response);
+});
+
+app.post("/todos/:id/done", (request, response) => {
+  myPackage.markAsDoneOrNotDone(request, response, true);
+});
+
+app.delete("/todos/:id/done", (request, response) => {
```

Comment:
Shouldn't be `put` method?

Reply:
`markAsDone (POST /todos/:id/done)
Sets the done flag of a single to-do to true

markAsNotDone (DELETE /todos/:id/done)
Sets the done flag of a single to-do to false`

These were the instructions. that's why I used `post` and `delete`. Maybe you're right and it should have been `put` but I'm not sure.